handbook/engineering/releases/upgrade_managed_issue_template.md

```diff
@@ -0,0 +1,17 @@
+<!--
+DO NOTE COPY THIS ISSUE TEMPLATE MANUALLY. Use `yarn run release tracking-issue:create` from the
```

Comment:
I'm guessing this will be `release:close` instead?

Reply:
I would argue it's better to create all the issues at the start of the release, even if we don't complete them before the other issue is closed

Reply:
Hm, in that case I wonder if we should just include this in the release process? e.g. make this a requirement to closing the release tracking issue

Reply:
sorry this was a bit **strong**. My rationale is that we open all tracking issues to start the release and close them as they're done. Each task of these issues are largely independent, as they are tied to the release being cut...not it being finished. 

I hope that makes more sense.

Reply:
makes sense!